twitch.go

```diff
@@ -180,13 +183,25 @@ func handleWebhook(w http.ResponseWriter, r *http.Request) {
 			cmd.Env = append(cmd.Env, "AUDIODEV=hw:1,0")
 			go cmd.Run()
 		} else if reward == "scrollo" {
-			f, err := os.Create("scrollo.txt")
+			scrolloFile := filepath.Join(".scrollocache/", params)
```

Comment:
love to spend my simpbucks to put my favourite uplifting message on the screen; “../keys”

brings a tear to my eye every time it does

Reply:
hashing it to fix this, not sure how i could have missed it